src/Types/CborBytes.purs

```diff
@@ -0,0 +1,79 @@
+-- | CborBytes. A wrapper over `ByteArray` to indicate that the bytes are cbor.
+
+module Types.CborBytes
+  ( CborBytes(..)
+  , cborBytesToByteArray
+  , cborBytesFromByteArray
+  , cborBytesToIntArray
+  , cborBytesFromIntArray
+  , cborBytesFromIntArrayUnsafe
+  , cborBytesToHex
+  , byteLength
+  , hexToCborBytes
+  , hexToCborBytesUnsafe
+  ) where
+
+import Data.Newtype (class Newtype, wrap, unwrap)
+
+import Types.ByteArray (ByteArray)
+import Types.ByteArray as BA
+import Data.Maybe (Maybe)
+import Prelude
+import Aeson (class DecodeAeson, decodeAesonViaJson)
+import Data.Argonaut (class DecodeJson, decodeJson)
+import Test.QuickCheck.Arbitrary (class Arbitrary, arbitrary)
+
+-- | An array of Bytes containing CBOR data
+newtype CborBytes = CborBytes ByteArray
+
+instance Show CborBytes where
+  show rb = "(cborBytesFromIntArrayUnsafe" <> show (cborBytesToIntArray rb) <>
```

Comment:
Might be worth adding a space after the initial string `cborBytesFromIntArrayUnsafe`.
Also, we've transitioned to use its hex representation for `Show`, as it's a little easier to debug with so I would suggest `cborBytesToHex` instead of `cborBytesToIntArray`

Reply:
Good catch! I'd say adding a space is important to make sure that the `Show` instance is a valid Purescript expression that will evaluate to the `show`n value